augur/export_v2.py

```diff
@@ -767,6 +768,21 @@ def set_build_url(data_json, config, cmd_line_build_url):
     elif config.get("build_url"):
         data_json['meta']['build_url'] = config.get("build_url")
 
+def set_description(data_json, cmd_line_description_file):
+    """
+    Read Markdown file provided by *cmd_line_description_file* and set
+    `meta.description` in *data_json* to the text provided.
+    """
+    if not cmd_line_description_file.endswith('.md'):
+        fatal("Provided description file needs to be a Markdown file.")
+
+    if not os.path.isfile(cmd_line_description_file):
+        fatal("Provided description file {} does not exist".format(cmd_line_description_file))
+
+    with open(cmd_line_description_file) as description_file:
+        markdown_text = ''.join(description_file.readlines())
```

Comment:
`description_file.read()` will read the whole file and return a single string. No need to join.